share/spice/expand_url/title_content.handlebars

```diff
@@ -0,0 +1,4 @@
+<h3 class="c-base__title text--primary">
+    <a href='http://{{org_url}}'>{{org_url}}</a>
+</h3>
+<h4 class="c-base__sub">{{context_info}}</h4>
```

Comment:
We can actually define the `subtitle` in the JS so the built-in template handles rendering it, but this is OK 👍